multiqc/modules/mosdepth/mosdepth.py

```diff
@@ -15,6 +15,49 @@
 log = logging.getLogger(__name__)
 
 
+def read_config():
+    cfg = getattr(config, "mosdepth_config", dict())
+    if type(cfg) != dict:
+        return {}
+
+    cfg["include_contigs"] = cfg.get("include_contigs", [])
+    if type(cfg["include_contigs"]) != list:
+        cfg["include_contigs"] = []
+
+    cfg["exclude_contigs"] = cfg.get("exclude_contigs", [])
+    if type(cfg["exclude_contigs"]) != list:
+        cfg["exclude_contigs"] = []
+
+    cfg["xchr"] = cfg.get("xchr", None)
+    if type(cfg["xchr"]) != str:
+        cfg["xchr"] = None
+
+    cfg["ychr"] = cfg.get("ychr", None)
+    if type(cfg["ychr"]) != str:
+        cfg["ychr"] = None
```

Comment:
Looking forward to using Pydantic for some of this stuff...!

Reply:
Oh for sure!